src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -154,8 +154,8 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
    * @returns The appropriate Corporation or Person name.
    */
   private formatName (filing: any): string {
-    return filing.orgName ? filing.orgName
-      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+    return filing?.person?.orgName ? filing?.person?.orgName
```

Comment:
FYI, this will return null if filing is null or person is null (or orgName is null). This is probably OK.